cddagl/ui.py

```diff
@@ -122,11 +161,64 @@ def arstrip(value):
         value = value[:-1]
     return value
 
-def is_64_windows():
-    return 'PROGRAMFILES(X86)' in os.environ
+def is_osx():
+    return platform.system() == 'Darwin'
+
+def is_windows():
+    return platform.system() == 'Windows'
+
+def is_linux():
+    return platform.system() == 'Linux'
+
+def is_posix():
+    return os.name == 'posix'
+
+def is_64_system():
+    return sys.maxsize > 2**32

```

Comment:
This will fail detecting a 64-bit OS that runs a 32-bit version of Python. This needs to be changed so it can detect the OS bitness.